src/dyld.rs

```diff
@@ -123,7 +123,7 @@ pub fn search_lists<T>(
         .iter()
         .flat_map(|&n| n)
         .find(|&(sym, _)| *sym == symbol)
-        .map(|&(_, ref f)| f)
+        .map(|(_, f)| f)
```

Comment:
Sorry, super nitpick, can you revert the change in this file? Just because it's nothing to do with your commit.

Reply:
Whoops :sweat_smile:, just run `cargo clippy --fix`